org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -1281,7 +1281,11 @@ class String {
 	 * 		"become".contains("CO")      ==> Answers false
 	 */
 	method contains(other) {
-		return self.indexOf(other) > 0
+		try {
+			return self.indexOf(other) >= 0
+		} catch e {
```

Comment:
I wonder if we should be catching the particular exception here. Otherwise it will be catching any error